source/SIL.AppBuilder.Portal.Frontend/src/lib/collection.ts

```diff
@@ -8,6 +8,15 @@ export function isEmpty(data) {
   );
 }
 
+// https://stackoverflow.com/a/28248573
+export function toSentence(arr, last = 'and') {
```

Comment:
this is only used in one place, for rendering an error / warning message where I need to concat a list of things together and make it not read awkward